TimePicker/TimePicker.js

```diff
@@ -25,7 +25,7 @@ import TimePickerBase from './TimePickerBase';
 /*
  * Converts a string representation of time into minutes
  *
- * @param	{String}	time	Time in the format `HH:mm`
+ * @param	{String}	        Time in the format `HH:mm`
```

Comment:
This is not a typo. It's the name of the parameter. It's the same at line 59. I think we should keep it like that.

Reply:
Done